crates/voicevox_core_python_api/python/voicevox_core/_models.py

```diff
@@ -6,27 +6,59 @@
 
 @pydantic.dataclasses.dataclass
 class StyleMeta:
+    """**スタイル** (_style_)のメタ情報。"""
+
     name: str
+    """スタイル名。"""
+
     id: int
+    """スタイルID。"""
 
 
 @pydantic.dataclasses.dataclass
 class SpeakerMeta:
-    """メタ情報。"""
+    """**話者** (*speaker*)のメタ情報。"""
 
     name: str
+    """話者名。"""
+
     styles: List[StyleMeta]
+    """話者に属するスタイル。"""
+
     speaker_uuid: str
+    """話者のバージョン。"""
+
     version: str
+    """話者のUUID。"""
 
 
 @pydantic.dataclasses.dataclass
 class SupportedDevices:
-    """サポートデバイス情報。"""
+    """
+    このライブラリで利用可能なデバイスの情報。
+
+    あくまで本ライブラリが対応しているデバイスの情報であることに注意。GPUが使える環境ではなかったとしても
+    ``cuda`` や ``dml`` は ``True`` を示しうる。
+    """
 
     cpu: bool
+    """
+    CPUが利用可能。
+
+    常に ``true`` 。
```

Comment:
```suggestion
    常に ``True`` 。
```

細かいですが, 横から失礼します…！  
おそらく下の doc に寄せられてしまったのかもですが, このファイルの記述は Python なので, `True` かと…！

https://github.com/qryxip/voicevox_core/blob/f9a1869b9b5c35f3c4d2e9747ab494886da2ce31/crates/voicevox_core/src/devices.rs#L13

Reply:
確かにそうですね。ありがとうございます。